capa/render/default.py

```diff
@@ -175,15 +175,23 @@ def render_maec(doc: rd.ResultDocument, ostream: StringIO):
         |                          | launcher                                                  |
         +--------------------------+-----------------------------------------------------------+
     """
-    maec_schema = collections.defaultdict(set)
+    maec_categories = {
```

Comment:
alternatively, consider to define a tuple with all the keys and do a `<entry>.replace("_", "-")`